README.md

```diff
@@ -47,23 +47,21 @@ Working on the indexing pipeline effectively requires internal access to the
 Internet Archive cluster and services, though some contributions and bugfixes
 are probably possible without staff access.
 
-To install dependencies for the first time, then run the tests (to ensure
-everything is working):
+To install dependencies for the first time run:
+    `make dep`
+then run the tests (to ensure everything is working):
+    `make test`
 
-    make dep
-    make test
-
-If developing the web interface, you will almost certainly need an example
+While developing the web interface, you will almost certainly need an example
 database running locally. A docker-compose file in `extra/docker/` can be used
 to run Elasticsearch 7.x locally. The `make dev-index` command will reset the
 local index with the correct schema mapping, and index any intermediate files
 in the `./data/` directory. We don't have an out-of-the-box solution for non-IA
 staff at this step (yet).
 
-After making changes to any user interface strings, the interface translation
```

Comment:
I'm not sure what the intent was with this change, but I think it only removed a period. If the intent was to have the "make" command show as separate code, it needs to have 4 spaces in front of it, and be separated from the text paragraph with blank lines. In this case, I think having the command in-line is appropriate though.

Reply:
Yeah, the objective was to get the `make` command to appear in a new line. I concur to having it inline; will change it right away